src/Orchard.Cms.Web/Modules/Orchard.Demo/Orchard.Demo.csproj

```diff
@@ -20,6 +20,7 @@
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.Environment.Navigation\Orchard.Environment.Navigation.csproj" />
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.ResourceManagement\Orchard.ResourceManagement.csproj" />
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.ResourceManagement.Abstractions\Orchard.ResourceManagement.Abstractions.csproj" />
+    <ProjectReference Include="..\Orchard.Contents\Orchard.Contents.csproj" />
```

Comment:
Maybe, we should think on to moving permissions to a an Abstractions versión of the module for all the permission clases in modules. It will be useful when you want a tenant as authorization server can provide permissions for anything to other tenants but you don't want it really implements the functionality of the services for which it is providing permissions.